tests/validation/README.md

```diff
@@ -0,0 +1,90 @@
+
+- `cc -o sender_app sender_app.c -lbsd $(pwd)/../../_build/lib/libmcm_dp.so`
+- `cc -o recver_app recver_app.c -lbsd $(pwd)/../../_build/lib/libmcm_dp.so`
+
+## Usage
+
+Remember to properly start the Media Proxy application before running the sample apps, in order to make the transmission possible.
+
+Use `app_name -H` to see full usage options for a specific application.
+
+### Common switches
+
+> **Note:** Kept here with a best-effort approach and may be obsolete. Check usage help (`-H` or `--help`) for the newest information about present switches.
+
+| Switch | Long switch       | Variable type      | Argument value         | Default                        | Examples                   |
+| ------ | ----------------- | ------------------ | ---------------------- | ------------------------------ | -------------------------- |
+| `-H`   | `--help`          | N/A                | empty (no argument)    | N/A                            | N/A                        |
+| `-b`   | `--file_name`     | `static char[128]` | Input/Output file name | `""` (empty)                   | `"in.yuv"` / `"out.yuv"`   |
+| `-w`   | `--width`         | `uint32_t`         | Video width [pixels]   | `1920`                         | `1920`                     |
+| `-h`   | `--height`        | `uint32_t`         | Video height [pixels]  | `1080`                         | `1080`                     |
+| `-f`   | `--fps`           | `double`           | Frames per second      | `30.0`                         | `25`                       |
+| `-x`   | `--pix_fmt`       | `char[32]`         | Video pixel format     | `"yuv422p10le"`                | `"yuv444p"`                |
+| `-r`   | `--rcv_ip`        | `char[253]`        | Receiver IP address    | `"127.0.0.1"`                  | `"192.168.96.1"`           |
+| `-i`   | `--rcv_port`      | `char[5]`          | Receiver port number   | `9001`                         | `9001`                     |
+| `-s`   | `--send_ip`       | `char[253]`        | Sender IP address      | `"127.0.0.1"`                  | `"192.168.96.2"`           |
+| `-p`   | `--send_port`     | `char[5]`          | Sender port number     | `9001`                         | `9002`                     |
+| `-o`   | `--protocol_type` | `char[32]`         | Type of protocol       | `"auto"`                       | `"memif"`                  |
+| `-t`   | `--payload_type`  | `char[32]`         | Type of payload        | `"st20"`                       | `"rdma"`/`"st22"`/`"st30"` |
+| `-k`   | `--socketpath`    | `char[108]`        | Socket path            | `"/run/mcm/mcm_rx_memif.sock"` | `"/other/path.sock"`       |
+| `-d`   | `--interfaceid`   | `uint32_t`         | ID of an interface     | `0`                            | `1`                        |
+| `-l`   | `--loop`          | `bool`             | Whether to loop or not | `0` (false)                    | `1` (true)                 |
+
+### Usage examples
```

Comment:
Add the step to setup the environment, where you enable the VF.

Reply:
Added a short instruction for VF creation.